core/messaging/README.md

```diff
@@ -0,0 +1,36 @@
+# @baseline-protocol/messaging
+
+Baseline core messaging package.
```

Comment:
On reflection, NATs is of course a good option for Baseline, but what if someone wants to use a different messaging infrastructure. What could be done to support this?

Reply:
Adding an implementation which would live as a sibling to the NATS and Whisper implementations. And also adding a provider type and supporting it within the factory.